ui/raidboss/data/04-sb/raid/o3n.js

```diff
@@ -0,0 +1,474 @@
+'use strict';
+
+// O3 - Deltascape 3.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V3\.0\)$/,
+  },
+  timelineFile: 'o3n.txt',
+  timelineTriggers: [
+    {
+      id: 'O3N Frost Breath',
+      regex: /Frost Breath/,
+      beforeSeconds: 4,
+      condition: function(data) {
+        return data.role == 'tank'&& !data.activeTank == data.me;
```

Comment:
I can't find an eslint rule for this, but can you add a space before the && operator?

Reply:
I'll be careful in future. That being said, this condition was probably unnecessary altogether here.